exercises/acronym/lib/example.dart

```diff
@@ -5,6 +5,8 @@ class Acronym {
     RegExp separatorsExp = new RegExp(r'( )|(-)');
     List<String> list = input.split(separatorsExp);
 
+    while (list.contains('')) list.remove('');
```

Comment:
Recommend to add a comment that explains why this step is necessary

Reply:
You could also simplify: `while (list.remove(' '));`

Reply:
@jvarness Will do.